lib/screen_controller.dart

```diff
@@ -147,7 +147,11 @@ class ScreenController {
         final res = await Navigator.push(
           context,
           MaterialPageRoute(builder: (context) => CameraScreen(
-            initialCamera: action.options!['initialCamera'] ?? 'DEFAULT',
+            mode: action.options!['mode'] != null ? Utils.getString(action.options!['mode'], fallback: '') : 'DEFAULT',
```

Comment:
Please never force !. This will throw an exception if options are not included. Options are comletely optional

Reply:
i fixed